gateway-ha/src/test/java/io/trino/gateway/ha/handler/TestQueryIdCachingProxyHandler.java

```diff
@@ -61,6 +66,44 @@ public void testExtractQueryIdFromUrl()
                 .isNull();
     }
 
+    @Test
+    public void testQueryIdFromKill()
```

Comment:
Remove `public`. https://trino.io/docs/current/develop/tests.html#conventions-and-recommendations
> Test methods should be defined as package-private.